nautobot_golden_config/views.py

```diff
@@ -66,7 +68,7 @@ class GoldenConfigBulkDeleteView(generic.BulkDeleteView):
 
     queryset = models.GoldenConfig.objects.all()
     table = tables.GoldenConfigTable
-    filterset = filters.GoldenConfigFilter
+    filterset = DeviceFilterSet
```

Comment:
Have you tested the bulk-delete view with this change? It looks a bit suspicious to me to be using DeviceFilterSet with a GoldenConfig queryset.